templates/types/streaming/nextjs/app/api/chat/engine/chat.ts

```diff
@@ -1,6 +1,6 @@
 import { Settings, SimpleChatEngine } from "llamaindex";
 
-export async function createChatEngine() {
+export async function createChatEngine(ids: string[]) {
```

Comment:
Why `ids` is added here but we don't use it?

Reply:
it's just an usued parameter for the factory